base/uuid.jl

```diff
@@ -1,5 +1,10 @@
 # This file is a part of Julia. License is MIT: https://julialang.org/license
 
+"""
+    Represents a Universally Unique Identifier (UUID).
+    Can be built from a UInt128 (all byte values), from two UInt64 or four UInt32.
```

Comment:
I'd suggest wrapping the types in backticks, and perhaps tweaking the phrasing to be more consistent when listing the three options -- e.g. "...from one `UInt128` (all byte values), from two `UInt64`, or from four `UInt32`." (or only use "from" once). Note the "a" --> "one", and the added comma after `UInt64`.

Reply:
I just re-read it, agreed on all points